backend/file-system/src/main/java/com/htec/filesystem/controller/FileController.java

```diff
@@ -80,6 +81,15 @@ public ResponseEntity zipDownload(@AuthenticationUser AuthUser user, @RequestBod
         return ResponseEntity.status(HttpStatus.OK).body(new TextResponseMessage(FILE_UPLOADED, HttpStatus.OK.value()));
     }
 
+    @PostMapping("/copy/{fileId}/{shelfId}/{userId}")
```

Comment:
maybe better to have something like /copy/file/{fileId}/shelf/{shelfId}/user/{userId} otherwise someone can easely switch the order